app/kumactl/cmd/install/testdata/install-control-plane.dump-values.yaml

```diff
@@ -236,10 +236,10 @@ controlPlane:
   # `Secret` is the name of the Secret,
   # and `Key` is the key of the Secret value to use
   secrets:
-  #  someConfig:
-  #    name: some-config
-  #    mountPath: /etc/some-config
-  #    readOnly: true
+  #  someSecret:
+  #    Secret: some-secret
+  #    Key: secret_key
+  #    Env: SOME_SECRET
```

Comment:
@slonka this is interesting because golden_files in your PR didn't fail because the yaml is the same.
It might be worth fixing this in a future PR

Reply:
Yeah, noticed the same in my PR. Let's make an issue for this.

Reply:
Acutally just made: https://github.com/kumahq/kuma/pull/5983